src/cdomains/addressDomain.ml

```diff
@@ -117,6 +115,7 @@ struct
       | false, false -> join x y
   *)
 
+  (* TODO: overrides is_top, but not top? *)
```

Comment:
I suppose it's weird because there isn't just a single value that this `is_top` returns true for. Probably good to have a comment about it to avoid surprises until we somehow design a more consistent domain where this inconsistency isn't necessary.